### PR TITLE
Core/Spells: Implement SPELL_AURA_KEYBOUND_OVERRIDE

### PR DESCRIPTION
**Pull request by Smerdokryl**
**Changes proposed:**

-  Store SpellIDs assigned by Keybound Override auras 
-  Activate spells on CMSG_KEYBOUND_OVERRIDE

**Tests performed:**

Builds, works in-game

### CODE REVIEW

File: src/server/game/Server/Packets/SpellPackets.cpp [hunk 1]
```diff
@@ -1053,4 +1053,9 @@ void TradeSkillSetFavorite::Read()
     _worldPacket >> RecipeID;
     IsFavorite = _worldPacket.ReadBit();
 }
+
+void WorldPackets::Movement::KeyboundOverride::Read()
```
Comment (MaxtorCoder) [inline src/server/game/Server/Packets/SpellPackets.cpp:1057]:
```suggestion
void KeyboundOverride::Read()
```

File: src/server/game/Spells/Auras/SpellAuraEffects.cpp [hunk 2]
```diff
@@ -6247,6 +6247,27 @@ void AuraEffect::HandleForceBreathBar(AuraApplication const* aurApp, uint8 mode,
...
+void AuraEffect::HandleKeyboundOverride(AuraApplication const* aurApp, uint8 mode, bool apply) const
+{
+    if (!(mode & AURA_EFFECT_HANDLE_REAL))
+        return;
+
+    Player* player = aurApp->GetTarget()->ToPlayer();
+    if (!player)
+        return;
+
+    int32 OverrideID = GetMiscValue();
```
Comment (MaxtorCoder) [inline src/server/game/Spells/Auras/SpellAuraEffects.cpp:6259]:
```suggestion
    int32 OverrideID = GetMiscValue();
    if (!OverrideID)
        return;
        
```

File: src/server/game/Server/Packets/SpellPackets.cpp (hunk 1 above)
Comment (Smerdokryl) [reply src/server/game/Server/Packets/SpellPackets.cpp]: Oops

File: src/server/game/Spells/Auras/SpellAuraEffects.cpp (hunk 2 above)
Comment (MaxtorCoder) [reply src/server/game/Spells/Auras/SpellAuraEffects.cpp:6259]: Oh actually, camelCase please, saw it just now

**Comment by Shauren** [comment]
What does this fix?

**Comment by Thordekk** [comment]
Need sql too 👍 
DROP TABLE IF EXISTS `spell_keybound_override`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `spell_keybound_override` (
  `ID` int(10) unsigned NOT NULL DEFAULT '0',
  `Function` text,
  `Type` tinyint(4) NOT NULL DEFAULT '0',
  `SpellID` int(11) NOT NULL DEFAULT '0',
  `Flags` int(11) NOT NULL DEFAULT '0',
  `VerifiedBuild` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`ID`,`VerifiedBuild`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

**Pull request merged**